mlrun/api/db/sqldb/db.py

```diff
@@ -849,7 +849,7 @@ def _get_link_artifacts_by_keys_and_uids(self, session, project, identifiers):
             .all()
         )
 
-    def del_artifact(self, session, key, tag="", project=""):
+    def del_artifact(self, session, key, tag="", project="", commit=True):
```

Comment:
This methods eventually will be removed, so for yourself to decide whether you maintain theirs type hinting etc... but for newer methods lets be strict as we go and lets add deprecation warning on these.

Reply:
not must to add the deprecation now, as we might decide on the final version of deprecation closer to the full feature emerges. but keep it as a TODO